src/friendly-iframe-embed.js

```diff
@@ -267,6 +279,12 @@ function mergeHtml(spec) {
     });
   }
 
+  // Load CSP
+  if (spec.cspEnabled) {
+    result.push('<meta http-equiv=Content-Security-Policy ' +
```

Comment:
@keithwrightbos, I wanted to confirm: what's the difference between doing this and `iframe.setAttribute('csp', 'script-src none; ...')`?

Reply:
Do browsers actually support that? We weren't sure. We found a W3C spec for it, but nothing else.

Reply:
Ok, apparently Chrome supports it, though it's documented nowhere and I had to browse through the Blink source code to confirm. (Why is Web platform documentation always like this.) No idea about Safari.

Reply:
Yeah. Support is spotty. So the current approach is definitely a good thing, albeit a bit messier to do. If you want, you can also do a switch on it with 
```
if ('csp' in iframe) {
  iframe.setAttribute('csp', ...);
} else {
  mergeIntoHTML(...);
}
```

One place where `csp` attribute wins is if we want to set any CSP property that's explicitly not allowed in the `meta` tag. In this case, `meta` will not work, but `csp` attribute should work. But I don't believe this applies to any of the current CSPs you use.

Reply:
The only directives disallowed in `<meta>` are `sandbox` (redundant), `disown-opener` (nonsensical for an iframe), `frame-ancestors` (we already control the embedding context), and `report-to` (strictly less powerful than an event listener). Every directive we'd ever want to use (i.e., fetch directives) should work fine.

Reply:
Perfect.